resources/js/store/user/api/userApi.ts

```diff
@@ -0,0 +1,23 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+export const userApi = createApi({
+  reducerPath: "userApi",
+  baseQuery: fetchBaseQuery({ baseUrl: "http://127.0.0.1:8000/api/" }),
```

Comment:
you can declare this in .env so that we can efficiently re use the base route from anywhere

Reply:
Done, moved to `env` file

Reply:
![image](https://user-images.githubusercontent.com/105616953/176084946-0243d222-d901-49eb-aff2-9e50edfd1ca8.png)